packages/manager/cypress/e2e/core/cloudpulse/dbaas-widgets-verification.spec.ts

```diff
@@ -260,6 +260,8 @@ describe('Integration Tests for DBaaS Dashboard ', () => {
 
     ui.autocompletePopper.findByTitle(clusterName).should('be.visible').click();
 
+    cy.get('body').click('topRight');
```

Comment:
just need to add a small comment for this click..
```suggestion
    cy.get('body').click('topRight'); // close database clusters dropdown
```